src/cbottle/models/networks.py

```diff
@@ -666,6 +681,10 @@ def __init__(
         temporal_attention: bool = False,
         time_length: Optional[int] = None,
         checkpoint: bool = True,
+        use_apex_groupnorm: bool | None = None,
+        is_healpix_factory: bool = False,
+        padding_backend: PaddingBackends | None = None,
```

Comment:
As opposed to the groupnorm implementation, we can already change the padding backend using the context manager, so no need to pass this argument further down into the implementations.

Reply:
good point. Thanks for investigating.